packages/sdk/src/types/runtime/collider.ts

```diff
@@ -8,13 +8,28 @@ import { InternalCollider } from '../internal/collider';
 import { CollisionHandler, TriggerHandler } from './physics';
 import { ColliderEventType, CollisionEventType, TriggerEventType } from './physics/collisionEventType';
 
+/**
+ * Controls what the assigned actors will collide with.
+ * * `default` - Good for most actors. These will collide with all "physical" things: other default actors,
+ * navigation actors, and the non-MRE environment. It also blocks the UI cursor and receives press/grab events.
+ * * `navigation` - For actors considered part of the environment. Can move/teleport onto these colliders,
+ * but cannot click or grab them.
+ * * `hologram` - For "non-physical" actors. Only interact with the cursor and other holograms.
+ * * `ui` - Actors in this layer do not collide with anything but the UI cursor.
+ */
+export type CollisionLayer

```

Comment:
I though that we were moving away from the type model in favor of enums for this sort of thing.  Is that not the case any longer?

Reply:
Good catch. I was remembering feedback I got on a previous PR to change to a type, but that was an interface, not an enum.

Reply:
I prefer enums personally. The primary reason for this preference is the ease of documentation. Individual values of a type cannot be documented with typedoc, but enum values can.

Reply:
To close the loop... Steven and I discussed the merits of `type` vs `enum` for use-cases like this one, and enums appear to be the better choice. I was swayed primarily by the fact that it isn't possible to add docstrings to individual elements of a union type (well, you can but it's unwieldy).

If no objection, going forward let's standardize on using "enum of string literals" for enum-like fields rather than "typed union of string literals".

I'll file an issue to track the work of fixing up the rest of the codebase. There will be some edge cases.